CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 ## HEAD
 
+## 0.17.2
+
+###### _Mar 27, 2017_
+
+A big thanks to the 16 contributors who are making this release possible.
```

Comment:
This is insignificant, but it would be more typical to use past-tense here: "A big thanks to the 16 contributors who made this release possible." (Their contributions are in the past, and so is this release by the time the release notes are being read.)